bundles/azure-service-broker-0.0.1/chart/azure-service-broker/values.yaml

```diff
@@ -0,0 +1,92 @@
+replicaCount: 1
+logLevel: "INFO"
+
+image:
+  ## Image location, NOT including the tag
+  repository: microsoft/azure-service-broker
+  ## Image tag
+  tag: v1.4.0
+  ## "IfNotPresent", "Always", or "Never"
+  pullPolicy: IfNotPresent
+
+## Whether to register this broker with the Kubernetes Service Catalog. If true,
+## the Kubernetes Service Catalog must already be installed on the cluster.
+## Marking this option false is useful for scenarios wherein one wishes to host
+## the broker in a separate cluster than the Service Catalog (or other client)
+## that will access it.
+registerBroker: true
+
+service:
+  ## Type of service; valid values are "ClusterIP", "LoadBalancer", and
+  ## "NodePort". "ClusterIP" is sufficient in the average case where a service
+  ## catalog installation in the same cluster is the only client that needs to
+  ## communicate with this broker. i.e. The broker does not need to receive
+  ## inbound requests from outside the cluster.
+  type: ClusterIP
+  ## Further configuration if service is of type "NodePort"
+  nodePort:
+    ## Available port in allowable range (e.g. 30000 - 32767 on minikube)
+    port: 30080
+
+# Secret name where azure credentials are keep
+# like "subscriptionId", "tenantId", "clientId", "clientSecret"
+secretCredentialName: azure-broker-data
+
+azure:
+  ## Valid values are "AzurePublicCloud" and "AzureChinaCloud"
+  environment: AzurePublicCloud
+
+tls:
+  enabled: false
```

Comment:
Original chart has true. Please revert and test.

Reply:
👍